pageserver/src/restore_local_repo.rs

```diff
@@ -173,7 +173,7 @@ fn import_relfile(
                     break;
                 }
                 _ => {
-                    error!("error reading file: {:?} ({})", path, e);
+                    error!("error reading file {:?}: {}", path, e);
```

Comment:
Small notes: here and in the next fixed place in the same file

*  `e` is also from `anyhow` and can be formatted with `{:#}`
* `path` with `{:?}` formatting will have escaped symbols and commas around. 
Not bad, just might be a bit unusual compared to `{}` formatting and `path.display()` (that instead does lossy conversion for any weird non-unicode cases here) usage instead that we have in many other places, can unify that.

Not saying you have to fix any of the bullets, just sharing the thoughts.

Reply:
Makes sense, changed this to `{:#}` for the error, and `{}` and `path.display()` for the path.

I also changed this into `bail!()` rather than `error!()`. If an error happens while reading a file, I think we should stop, not continue importing the rest of the cluster as if nothing happened.